flake.nix

```diff
@@ -30,7 +30,6 @@
             pkgs.elmPackages.elm
             pkgs.elmPackages.elm-format
             pkgs.elmPackages.elm-test
-            pkgs.elmPackages.elm-language-server
```

Comment:
In monorepo we removed the elm LS from the nix configuration so I figured we should here as well. The LS is managed via the vscode extension, or otherwise its recommended to install at the user level.